src/main/java/evaluator/App.java

```diff
@@ -31,31 +34,80 @@
 
   @Parameters(index = "1", description = "Template file")
   private File templateFile;
+  
+  @Parameters(index = "2", description = "Folder for URL cache")
+  private String URLCacheFolderName;
+  
+  private boolean URLCacheEnabled = true;
+  private File URLCache;
 
   public static void main(String[] args) {
     CommandLine.call(new App(), args);
   }
-
+ 
   @Override
-  public Void call() throws Exception {
+  public Void call() throws Exception {    
+	try {
```

Comment:
Extract this block into a function? initCache() or similar?

Reply:
Yes, good idea. Done. Actually, I also extracted all the cache functionality into a new class, HTMLCache. I'll submit a new Pull request with the changes, because there are a lot more changes now.